### DISCUSSION TITLE
Calendar layout: Subtitle in the display template

### DISCUSSION THREAD

**Discussion by abdonrd** [directus/directus#12244] [Feature Requests]
I want to propose to add a subtitle option to the Calendar layout.
Like in the Cards layout.

As the calendar events have very small space, a second line would be a great help.

**Comment by rijkvanzanten** [comment]
Makes sense to me! Lets see if [fullcalendar supports that](https://fullcalendar.io/docs/month-view)  by default

**Reply by abdonrd** [reply]
Seems like we should add a `<br>` and thats all.

https://github.com/fullcalendar/fullcalendar/issues/450#issuecomment-1016726634

I test it with the Chrome Dev Tools and it works.

**Reply by rijkvanzanten** [reply]
oh jeez, sometimes the solutions are so obviously simple that it doesn't even cross my mind haha

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
